app/models/firm.rb

```diff
@@ -12,6 +12,11 @@ class Firm < ActiveRecord::Base
   ]
 
   scope :registered, -> { where.not(email_address: nil) }
+  scope :sorted_by_registered_name, -> { order(:registered_name) }
+
+  def self.most_recently_updated(limit: nil)
+    order(updated_at: :desc).limit(limit)
+  end
```

Comment:
Me neither. Currently I'm seeing the limit as an application side decision, let's see how it goes.